.github/workflows/docker-builder-packaging-plugins.yml

```diff
@@ -6,16 +6,16 @@ concurrency:
 
 on:
   workflow_dispatch:
-#  push:
-#    branches:
-#      - develop
-#    paths:
-#      - ".github/workflows/docker-builder-packaging-plugins.yml"
-#      - ".github/docker/*"
-#  pull_request:
-#    paths:
-#      - ".github/workflows/docker-builder-packaging-plugins.yml"
-#      - ".github/docker/*"
+  push:
+    branches:
+      - develop
+    paths:
+      - ".github/workflows/docker-builder-packaging-plugins.yml"
+      - ".github/docker/*"
+  pull_request:
+    paths:
+      - ".github/workflows/docker-builder-packaging-plugins.yml"
+      - ".github/docker/*"
```

Comment:
Should we not remove run triggers on workflow file changes like we are doing on the other repos?

Reply:
it is the workflow to build packaging docker, so there is no issue for releases